frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -56,15 +64,51 @@ export const BrowseModels = ({
 
   if (modelsFiltered.length) {
     return (
-      <GridContainer role="grid">
-        {groupsOfModels.map(groupOfModels => (
-          <ModelGroup
-            models={groupOfModels}
-            key={`modelgroup-${groupOfModels[0].collection.id}`}
-            localeCode={localeCode}
-          />
-        ))}
-      </GridContainer>
+      <>
+        {shouldShowBanner && (
+          <Paper
+            mt="1rem"
+            p="1rem"
+            color="text-dark"
+            bg="brand-lighter"
+            shadow="0"
+            radius="0.25rem"
+            role="complementary"
+          >
+            <Flex>
+              <BannerModelIcon>
+                <Icon name="model" />
+              </BannerModelIcon>
+              <Group grow>
+                <Text size="md" lh="1rem" mr="1rem">
+                  {t`Models help curate data to make it easier to find answers to questions all in one place.`}
```

Comment:
```suggestion
                  {t`Create models to clean up raw data, join tables with related info, and make the data easier to query.`}
```

@kdoh what do you think about this edit? "Curate" is doing a lot of heavy lifting, so I wanted to spell out what we mean. Given how abstract models are, the elaboration might be worth it.

Also: this edit subtly implies that 1) _you_ can create models and 2) that _other people you work with_ are creating these models for their team(s). The original construction could make it seem like models just happen (or come from somewhere else, like raw tables), or that Metabase is curating them, a la X-rays or something.

As an aside, I'm not a fan of UI, like banners, that make me do work I didn't seek out, even if that work is a single click, one time. Do we really need to communicate this info about Models in a banner? Is this flash of context worth the risk of annoying people? Could this text be relegated to a tooltip? Or a subtitle beneath the tab?